### PR TITLE
Update ember times template

### PR DESCRIPTION
**Pull request by amyrlam**
## What it does
- Fix typo
- Add date
- ~Change tag to ember-times (could do thruout if ok)~
- Remove READMORE
- Add 2021 tag

- [x] Rebase https://github.com/ember-learn/ember-blog/pull/848 after this is merged

### CODE REVIEW

File: post-templates/embertimes-template.md [hunk 1]
```diff
@@ -2,18 +2,17 @@
 title: The Ember Times - Issue No. XXS
 author:
   - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-XX-XXT00:00:00.000Z
```
Comment (amyrlam) [inline post-templates/embertimes-template.md:5]: Is the date format ok? Always prefix `T00:00:00.000Z`?

File: post-templates/embertimes-template.md [hunk 2]
```diff
@@ -2,18 +2,17 @@
...
 tags:
-  - ewsletter
-  - ember-js-times
+  - newsletter
+  - ember-times
```
Comment (amyrlam) [inline post-templates/embertimes-template.md]: It seems like we dno't really use `ember-times` but if we do, remove the `js`?

File: post-templates/embertimes-template.md (hunk 1 above)
Comment (ijlee2) [reply post-templates/embertimes-template.md:5]: Yep. I think we can use https://raw.githubusercontent.com/ember-learn/ember-blog/master/content/the-ember-times-issue-170.md as a template.

File: post-templates/embertimes-template.md (hunk 2 above)
Comment (ijlee2) [reply post-templates/embertimes-template.md]: Maybe skip it based on the issue 170?
```suggestion
```

**Review by ijlee2 — APPROVED** [review]
Looks good to me, thanks for working on updating the template!

**Review by mansona — APPROVED** [review]
Looks great! sorry I missed some of the finer points in this one 🙈

**Comment by amyrlam** [comment]
@mansona no apologies needed, thanks for all of your hard work!

**Pull request merged**